app/controllers/admin/users_controller.rb

```diff
@@ -21,7 +21,7 @@ def update
 
   # DELETE /admin/users/1
   def destroy
-    @user.destroy
+    @user.enrollments.where(team: current_team).delete_all
```

Comment:
`destroy` and `delete` are different, `delete_all` will not run callbacks. This change is intentional?